src/routers/alpha-router/alpha-router.ts

```diff
@@ -843,12 +842,10 @@ export class AlphaRouter
       { blockNumber }
     );
 
-    const { protocols } = routingConfig;
+    const protocols = routingConfig.protocols ?? [];
```

Comment:
We can just create a set right away here
`const protocolsSet = new Set(routingConfig.protocols ??  []);` and just log the protocolsSet

Reply:
I thought so too, but we log the `protocols` as an Array, logging it as a set would change the structure of the log.

Actually, do you know if there's any particular reason of why we create a set? We could accomplish the same thing by doing `.includes` on the Array instead of `.has` on the Set

Reply:
I think just to auto filter out any accidental duplicates?

Reply:
yup, I ended up applying some logic to get the best of both worlds